### PR TITLE
fix: Fixed to handle the circular position

### PR DESCRIPTION
**Pull request by younkue**
## Issue

#124

### CODE REVIEW

File: src/AnimationManager.ts [hunk 1]
```diff
@@ -136,21 +136,25 @@ export class AnimationManager {
...
+				const isCanceled = !self.em.triggerChange(toPos, prevPos);
 
+				prevPos = map(toPos, v => toFixed(v));
 				if (easingPer >= 1) {
-					if (!equal(param.destPos, self.axm.get(Object.keys(param.destPos)))) {
-						self.em.triggerChange(param.destPos);
+					const destPos = param.destPos;
+
+					if (!equal(destPos, self.axm.get(Object.keys(destPos)))) {
+						self.em.triggerChange(destPos, prevPos);
```
Comment (jongmoon) [inline src/AnimationManager.ts]: prevPos 를 넘기는 대신에 info.delta 값을 넘기면 되지 않을까요? toPos 는 그대로 두고요.

File: src/InputObserver.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ export class InputObserver implements IInputTypeObserver {
 			event,
 		};
 		if (isEqual || userWish.duration === 0) {
-			!isEqual && this.em.triggerChange(userWish.destPos, changeOption, true);
+			!isEqual && this.em.triggerChange(userWish.destPos, depaPos, changeOption, true);
```
Comment (jongmoon) [inline src/InputObserver.ts]: 여기서도 위에서 delta 값을 계산하고 넘기면 어떨까요?

File: src/AnimationManager.ts (hunk 1 above)
Comment (younkue) [reply src/AnimationManager.ts]: @jongmoon 
DepaPos is the starting point
TestPos is the arrival point.
Delta is the difference between the arrival point and the starting point.

A temporary delta cannot be obtained through that delta.

File: src/InputObserver.ts (hunk 2 above)
Comment (younkue) [reply src/InputObserver.ts]: @jongmoon 
The destPos and depaPos do not contain all the information from the Axis.

Delta, which contains all the information, is obtained through the Axis.moveTo method in the 'triggerChange' method.

**Pull request merged**